snapcraft-options/snapcraft-agent.yaml

```diff
@@ -80,7 +81,19 @@ slots:
     content: port-number
     read: [ $SNAP_DATA/var/run/deliveryoptimization-agent ]
 
+  # Share a folder containing sdk-config.json
   config-file:
     interface: content
     content: config-file
-    write: [ $SNAP_DATA/etc/deliveryoptimization-agent/sdk-config.json ]
+    write: [ $SNAP_DATA/etc/deliveryoptimization-agent/configs ]
```

Comment:
Revert the sdk-config.json change. I will do it as part of the other path fixes for DO agent.

Reply:
This change has not been reverted.